features/step_definitions/user_can_create_account_assertions_steps.rb

```diff
@@ -0,0 +1,5 @@
+
+Then("I should see {string}") do |content|
```

Comment:
rename this file to `assertions_steps` instead

Reply:
Resolved, name changed